cip.go

```diff
@@ -186,14 +209,11 @@ func main() {
 	}
 	sc.ReadRepository(mkReadRepositoryCmd)
 
-	if *genImagesPtr {
-		if err := sc.GenImages(mfest); err != nil {
-			fmt.Fprintln(
-				os.Stderr,
-				fmt.Errorf("cannot create manifest: %s", err))
-			os.Exit(1)
-		}
-
+	if len(*snapshotPtr) > 0 {
+		var rii reg.RegInvImage
+		rii = sc.Inv[mfest.Registries[0].Name]
```

Comment:
Nit: `rii := ...` is easier than declaring the var and then immediately assigning it

Reply:
Heh, I keep forgetting about `:=`. Ack